x-pack/plugins/apm/public/services/rest/apm/metrics.ts

```diff
@@ -6,25 +6,26 @@
 
 import { MetricsChartAPIResponse } from '../../../../server/lib/metrics/get_all_metrics_chart_data';
 import { callApi } from '../callApi';
-import { getEncodedEsQuery } from './apm';
+import { getEncodedEsFilters } from '../../filters/get_encoded_es_filters';
+import { UIFilters } from '../../../../typings/ui-filters';
 
 export async function loadMetricsChartDataForService({
   serviceName,
   start,
   end,
-  kuery
+  uiFilters
 }: {
   serviceName: string;
   start: string;
   end: string;
-  kuery: string | undefined;
+  uiFilters: UIFilters;
 }) {
   return callApi<MetricsChartAPIResponse>({
     pathname: `/api/apm/services/${serviceName}/metrics/charts`,
     query: {
       start,
       end,
-      esFilterQuery: await getEncodedEsQuery(kuery)
+      uiFiltersES: await getEncodedEsFilters(uiFilters)
```

Comment:
I think we can be more spot on with the naming here. We call it `uiFilters` throughout the client-side. That seems good.
Then we call it `uiFiltersES` when we convert it. I guess that's ok. The method to convert between the two is called `getEncodedEsFilters` and `getEnvironmentESFilter` - shouldn't the naming have something with `uiFiltersES` if that's what it returns? 

Perhaps it should be called `getUiFiltersES` instead of `getEncodedEsFilters`?

Tbh. I'm not sure. Just don't feel it is fully consistent.

Reply:
I attempted to make it a bit more consistent to your suggestion, following a naming pattern of `get{type}UiFiltersES` and the snake-case equivalent for file names.

Reply:
Looks good, thanks 👍